src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const accessibilityQuiz = [
+  {
+    Question: 
+      "What does ASAP in the context of web development stand for?",
+    Answer: "As Semantic As Possible",
+    Distractor1: "As Soon As Possible",
+    Distractor2: "Automated System Application Protocol",
+    Distractor3: "Advanced Syntax Application Programming",
+    Explanation: 
+      "In the context of web development, ASAP stands for 'As Semantic As Possible', emphasizing the importance of semantic markup for accessibility and SEO.",
+    Link: "https://www.freecodecamp.org/news/4-principles-for-getting-started-with-website-accessibility-4e85f75730b/t"
```

Comment:
this links to a 404 page

please find another freeCodeCamp resource or w3.org resource to place here instead